CHANGELOG.md

```diff
@@ -1,8 +1,30 @@
-# dbt_google_ads version.version
+# dbt_google_ads v0.12.0
+
+## Schema Updates
+
+**3 total changes • 0 possible breaking changes**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| [google_ads__search_term_report](https://fivetran.github.io/dbt_google_ads/#!/model/model.google_ads.google_ads__search_term_report)  | New Transform Model |   |   |  Uses new `search_term_keyword_stats` source table  |
```

Comment:
The link looks good to me. Maybe we change up the `Notes` section by adding the description of the transform model. The staging models look fine.